src/parsers/forecast_parser.jl

```diff
@@ -1,16 +1,60 @@
+
+const CATEGORY_STR_TO_COMPONENT = Dict{String, DataType}(
+    "Bus" => Bus,
+    "Generator" => Generator,
+    "Reserve" => Service,
+    "LoadZone" => LoadZones,
+    "ElectricLoad" => ElectricLoad,
+)
+
+"""Describes how to construct forecasts from raw timeseries data files."""
+mutable struct TimeseriesMetadata
```

Comment:
This is cool! I would name it something that is more descriptive of the purpose like `TimeSeriesFilesMetaData` or something that uses the word files.

Reply:
Fixed.